### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -209,7 +209,68 @@ Enables you to limit the returned data by filtering on specific attributes. For
 
 The Filter parameter uses the same filtering syntax as the Where-Object cmdlet. For example, the following filter returns only users who have been enabled for Enterprise Voice: `-Filter 'EnterpriseVoiceEnabled -eq $True'` or ``-Filter "EnterpriseVoiceEnabled -eq `$True"``.
 
-The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later:
+**Updates in Teams PowerShell Module version 5.0.0**
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:212]:
```suggestion
**Teams PowerShell Module version 5.0.0 updates**
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -209,7 +209,68 @@ Enables you to limit the returned data by filtering on specific attributes. For
...
+- WhenCreated
+- FeatureTypes 
+- PreferredDataLocation
+- LastName
+
+_New Operators_
+
+These filtering operators have been reintroduced:
+
+- “-like” operator now supports use of wildcard operators in ‘contains’ and ‘ends with’ scenarios. Example:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:253]:
```suggestion
- “-like” operator now supports the use of wildcard operators in ‘contains’ and ‘ends with’ scenarios. For example:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -209,7 +209,68 @@ Enables you to limit the returned data by filtering on specific attributes. For
...
+
+```
+Contains Scenario: Get-CsOnlineUser  -Filter "AssignedPlan -like '*MCO*'"
+Ends with scenario: Get-CsOnlineUser  -Filter {AssignedPlan -like '*MCO'}
+```
+- “-contains” can now be used for filtering on properties that are an array of strings like FeatureTypes, ProxyAddresses and ShadowProxyAddresses. Example:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:259]:
```suggestion
- “-contains” can now be used to filter properties that are an array of strings like FeatureTypes, ProxyAddresses, and ShadowProxyAddresses. For example:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 4]
```diff
@@ -209,7 +209,68 @@ Enables you to limit the returned data by filtering on specific attributes. For
...
+```
+Get-CsOnlineUser -Filter {FeatureTypes -contains "PhoneSystem"}
+Get-CsOnlineUser -Filter {ProxyAddresses -contains "SMTP:abc@xyz.com"}
+```
+- “-gt”(greater than), “-lt”(less than), “-le”(less than or equal to) can now be used for filtering all string properties. Example:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:264]:
```suggestion
- “-gt” (greater than), “-lt” (less than), “-le” (less than or equal to) can now be used for filtering all string properties. For example:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 5]
```diff
@@ -209,7 +209,68 @@ Enables you to limit the returned data by filtering on specific attributes. For
...
+```
+Get-CsOnlineUser -Filter {UserPrincipalName -gt/-le/-lt “abc”}
+```
+- “-ge”(greater than or equal to) can now also be used for filtering on policies
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:268]:
```suggestion
- “-ge”(greater than or equal to) can now also be used for filtering on policies. For example:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 6]
```diff
@@ -209,7 +209,68 @@ Enables you to limit the returned data by filtering on specific attributes. For
...
+```
+Get-CsOnlineUser -Filter {ExternalAccessPolicy -ge "xyz_policy"}
+```
+
+The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later (excluding updates mentioned above for Teams PowerShell Module version 5.0.0):
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:273]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later (excluding updates mentioned previously for Teams PowerShell Module version 5.0.0):
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 7]
```diff
@@ -472,7 +534,24 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ### Notes
 
-The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later (note that these changes are only rolled out in commercial environments at present and will be applicable to the lastest TPM versions in government environments starting March 31, 2023.):
+**Updates in Teams PowerShell Module version 5.0.0**
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:537]:
```suggestion
**Teams PowerShell Module version 5.0.0 updates**
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 8]
```diff
@@ -472,7 +534,24 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+New attributes have now been introduced in the output of Get-CsOnlineUser when not using the "-identity" parameter:
+
+- CountryAbbreviation                   
+- SipProxyAddress                       
+- TeamsMediaLoggingPolicy               
+- UserValidationErrors                   
+- WhenCreated    
+     
+Unlicensed Users – Unlicensed user would show up in the output for 30 days post license removal 
+Soft deleted users – These users will be displayed in the output of Get-CsOnlineUser and TAC Manage Users page by default with SoftDeletionTimestamp set to a value
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:550]:
```suggestion
Unlicensed Users: Unlicensed users would show up in the output for 30 days post-license removal.
Soft deleted users: These users will be displayed in the output of Get-CsOnlineUser and the TAC Manage Users page by default with SoftDeletionTimestamp set to a value.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 9]
```diff
@@ -472,7 +534,24 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+
+**Updates in Teams PowerShell Module version 3.0.0 and above**
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:552]:
```suggestion
**PowerShell Module version 3.0.0 and above updates**
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 10]
```diff
@@ -472,7 +534,24 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+
+The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later excluding updates mentioned above for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments at present and will be applicable to the lastest TPM versions in government environments starting March 31, 2023.):
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:554]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later, excluding updates mentioned previously for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments at present and will be applicable to the lastest TPM versions in government environments starting March 31, 2023.):
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

**Comment by isrumnon** [comment]
> Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

Done!

**Comment by JohanFreelancer9** [comment]
> > Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.
> 
> Done!

@isrumnon, you seem to have missed one suggestion. Could you also please commit it?

**Comment by isrumnon** [comment]
> > > Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.
> > 
> > 
> > Done!
> 
> @isrumnon, you seem to have missed one suggestion. Could you also please commit it?

@JohanFreelancer9 Whoops, done now

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 11]
```diff
@@ -472,7 +534,24 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ### Notes
 
-The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later (note that these changes are only rolled out in commercial environments at present and will be applicable to the lastest TPM versions in government environments starting March 31, 2023.):
+**Updates in Teams PowerShell Module version 5.0.0**
+
+The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0  (note that these changes are only rolled out in commercial environments at present:
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:539]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0  (note that these changes are only rolled out in commercial environments at present):
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 12]
```diff
@@ -209,7 +209,68 @@ Enables you to limit the returned data by filtering on specific attributes. For
 
 The Filter parameter uses the same filtering syntax as the Where-Object cmdlet. For example, the following filter returns only users who have been enabled for Enterprise Voice: `-Filter 'EnterpriseVoiceEnabled -eq $True'` or ``-Filter "EnterpriseVoiceEnabled -eq `$True"``.
 
-The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later:
+**Updates in Teams PowerShell Module version 5.0.0**
+
+The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0  (note that these changes are only rolled out in commercial environments at present:
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:214]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 5.0.0  (note that these changes are only rolled out in commercial environments at present):
```

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**